ci-scripts/test_agent/Tester.py

```diff
@@ -59,27 +60,36 @@ def start_test(self, workload, build, test_done_callback):
         self.current_build = build
         self.callback = test_done_callback
 
-        # start test on current_workload
-        print("Tester {} Starting test on workload".format(self.id))
-        print(workload.key(), "==>", workload.val())
-
-        # register callback to call_ended()
-        # TODO pass pickle file from here to test run so we have control over it.
-
         def run_hil_thread(call_ended, popen_args):
-            proc = subprocess.Popen(*popen_args, stdout=subprocess.PIPE)
+            proc = subprocess.Popen(*popen_args, stdout=subprocess.PIPE, shell=True)
             proc.wait()
             call_ended()
             return
 
-        thread = threading.Thread(
-            target=run_hil_thread, args=(self.test_ended, ["./run_test.sh"])
-        )
-        thread.start()
+        if self.current_build.val()["agw"]["valid"]:
+            magma_build = self.current_build.val()["agw"]["artifacts"]["downloadUri"]
```

Comment:
If you can switch to using build['agw']['packages'] that would be great. This way we can quickly deprecate the old field
See example from latest builds on https://magma-ci.web.app/